finesse/hardware/temperature/seneca.py

```diff
@@ -0,0 +1,154 @@
+"""This module provides an interface to Seneca temperature readers."""
+from decimal import Decimal
+
+import numpy
+from serial import Serial, SerialException
+
+from .temperature_monitor_base import TemperatureMonitorBase
```

Comment:
This can be fixed by:

```suggestion
from finesse.hardware.temperature.temperature_monitor_base import TemperatureMonitorBase
```
I guess this is related to #378